protobufs/livekit_rtc.proto

```diff
@@ -223,6 +227,20 @@ message UpdateTrackSettings {
   uint32 priority = 8;
 }
 
+
+
+message UpdateLocalAudioTrack {
+  string track_sid = 1;
+  repeated AudioTrackFeature features = 2;
+}
+
+message UpdateLocalVideoTrack {
+  string track_sid = 1;
+  uint32 width = 2;
+  uint32 height = 3;
+  repeated VideoTrackFeature features = 4;
```

Comment:
If this is simulcast, may be have `width/height` per layer?

Reply:
tbh, I'm not sure if `simulcast` is even desired at this point. Is it something that we want to be able to turn on/off during a publications' lifecycle? 
cc @davidzhao

Reply:
I think changing publication (any feature) mid-stream is a discussion worth having. There are things that cannot be changed without re-acquiring a stream (things like AGC/echo cancellation for audio).

For simulcast specific, on the wall on that. Ideally, we don't even have to explicitly specify `simulcast` flag/feature. Based on number of layers, it can be implicit. 

But, there is a need to distinguish between VP9/AV1 Simulcast vs SVC. If/when we support VP9/AV1 simulcast, we will need to indicate if it is simulcast or SVC.  So, maybe, we can continue to have that flag. If VP9 simulcast, that flag can indicate it is simulcast, else treated as SVC.

Reply:
to keep it simple I removed (after discussing with dz) the `VideoFeature` part from the PR. 
For the beginning mostly resolution updates will be relevant for video tracks. We can add other things like layers later on if we see the need for it.